src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-remove-node.ts

```diff
@@ -56,142 +70,126 @@ class DialogZWaveJSRemoveNode extends LitElement {
-          : nothing}
-        ${this._status === "started"
-          ? html`
-              <div class="flex-container">
-                <ha-spinner></ha-spinner>
-                <div class="status">
-                  <p>
-                    <b
-                      >${this.hass.localize(
-                        "ui.panel.config.zwave_js.remove_node.controller_in_exclusion_mode"
-                      )}</b
-                    >
-                  </p>
-                  <p>
-                    ${this.hass.localize(
-                      "ui.panel.config.zwave_js.remove_node.follow_device_instructions"
-                    )}
-                  </p>
-                </div>
-              </div>
-              <mwc-button slot="primaryAction" @click=${this.closeDialog}>
-                ${this.hass.localize(
-                  "ui.panel.config.zwave_js.remove_node.cancel_exclusion"
-                )}
-              </mwc-button>
-            `
-          : nothing}
-        ${this._status === "failed"
-          ? html`
-              <div class="flex-container">
-                <ha-svg-icon
-                  .path=${mdiCloseCircle}
-                  class="failed"
-                ></ha-svg-icon>
-                <div class="status">
-                  <p>
-                    ${this.hass.localize(
-                      "ui.panel.config.zwave_js.remove_node.exclusion_failed"
-                    )}
-                  </p>
-                  ${this._error
-                    ? html`<ha-alert alert-type="error">
-                        ${this._error}
-                      </ha-alert>`
-                    : nothing}
-                </div>
-              </div>
-              <mwc-button slot="primaryAction" @click=${this.closeDialog}>
-                ${this.hass.localize("ui.common.close")}
-              </mwc-button>
-            `
-          : nothing}
-        ${this._status === "finished"
-          ? html`
-              <div class="flex-container">
-                <ha-svg-icon
-                  .path=${mdiCheckCircle}
-                  class="success"
-                ></ha-svg-icon>
-                <div class="status">
-                  <p>
-                    ${this.hass.localize(
-                      "ui.panel.config.zwave_js.remove_node.exclusion_finished",
-                      { id: this._node!.node_id }
-                    )}
-                  </p>
-                </div>
-              </div>
-              <mwc-button slot="primaryAction" @click=${this.closeDialog}>
-                ${this.hass.localize("ui.common.close")}
-              </mwc-button>
-            `
-          : nothing}
-      </ha-dialog>
+        ${this.hass.localize(
+          this._step === "start"
+            ? "ui.panel.config.zwave_js.remove_node.start_exclusion"
+            : this._step === "exclusion"
+              ? "ui.panel.config.zwave_js.remove_node.cancel_exclusion"
+              : "ui.common.close"
+        )}
```

Comment:
I don't like this but it's not a big deal

Reply:
okay, how would you like it :slightly_smiling_face: